pkg/storage/utils/decomposedfs/upload.go

```diff
@@ -297,16 +297,6 @@ func (fs *Decomposedfs) NewUpload(ctx context.Context, info tusd.FileInfo) (uplo
 		ctx:      ctx,
 	}
 
-	if !info.SizeIsDeferred && info.Size == 0 {
```

Comment:
I'm not sure if this would work as we skip sending the patch request for zero-size files https://github.com/cs3org/reva/blob/master/internal/http/services/owncloud/ocdav/tus.go#L262

I think this should be covered by the integration tests

Reply:
Looks like it isn't covered by tests since the CI is green. 
I don't really like the magic skip of zero byte uploads. Therefore i proposed: https://github.com/cs3org/cs3apis/pull/154.

Reply:
I removed the skipping of zero byte uploads for now. That's something for a different PR.